demos/crm-sharing/postgres-to-salesforce/deploy-serverless-app.md

```diff
@@ -68,10 +68,9 @@ The **salesforce** node needs to be configured to send block notifications to yo
 
 ```graphql
 mutation update_blockReportLambdas($notificationFunction: String!) {
-  updateVendia_Settings_async(
+  updateVendia_Settings(
     input: {aws: {blockReportLambdas: [$notificationFunction]}}
   ) {
```

Comment:
I think syncMode is required for this mutation

Reply:
Good topic to cover.  The default (`NODE_COMMITTED`) will be used if no `syncMode` is specified.  See - https://www.vendia.net/docs/share/graphql#mutations - for some examples and more on the defaults.